src/Compilers/VisualBasic/Portable/BoundTree/Statement.vb

```diff
@@ -1460,7 +1461,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 Return s_variablesMappings.GetValue(
                     Me,
                     Function(BoundUsing)
-                        Return New Variables(BoundUsing.ResourceList.As(Of IVariableDeclaration))
+                        Dim usingStatementSyntax = Syntax.ChildNodes().OfType(Of UsingStatementSyntax).SingleOrDefault()
+                        Return New Variables(BoundUsing.ResourceList.As(Of IVariableDeclaration), usingStatementSyntax)
```

Comment:
Of course, there is a risk that there is a consumer that doesn't handle null from IUsingStatement.Declaration.

---
In reply to: [119527163](https://github.com/dotnet/roslyn/pull/19885#discussion_r119527163) [](ancestors = 119527163)